### PR TITLE
Add 6.5 branch

### PR DESCRIPTION
**Pull request by JakobDev**
This tries to add a 6.5 branch. I have edited the patch files local with a text editor (I don't know how to properly update them), so they don't throw a error any more., but I have no Idea what they do and I can't tell, if that was done right. I can't even say if this this even builds. If this builds and works, this needs to be merged in a new branch.

### CODE REVIEW

**Comment by JakobDev** [comment]
> QtWebEngine will not be built: Unmodified ffmpeg >= 5.0 is not supported.

Does that mean I need to include a older ffmpeg version?

File: io.qt.qtwebengine.BaseApp.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
             "config-opts": [
                 "-DQT_FEATURE_webengine_kerberos=ON",
                 "-DQT_FEATURE_webengine_proprietary_codecs=ON",
-                "-DQT_FEATURE_webengine_system_ffmpeg=ON",
+                "-DQT_FEATURE_webengine_system_ffmpeg=OFF",
```
Comment (tsdgeos) [inline io.qt.qtwebengine.BaseApp.json:26]: why off?

Comment (JakobDev) [reply io.qt.qtwebengine.BaseApp.json:26]: The ffmpeg version of the runtime is not compatible with Chromium

Comment (tsdgeos) [reply io.qt.qtwebengine.BaseApp.json:26]: and it is in the 6.4 branch? That's odd, what's the error?

Comment (Erick555) [reply io.qt.qtwebengine.BaseApp.json:26]: This was explained [here](https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/issues/1611). There are 3 possible solutions:

1. runtime patches ffmpeg
2. baseapp patches webengine
3. internal ffmpeg is used

Comment (hfiguiere) [reply io.qt.qtwebengine.BaseApp.json:26]: The thing is that 6.4 and 6.5 use the same freedesktop runtime. So if it works in 6.4 then it works in 6.5

Which is either 6.4 is broken or this should just be the same as 6.4

You can't expect that change in freedesktop runtime as 6.5 will not use a more recent runtime.

Comment (tsdgeos) [reply io.qt.qtwebengine.BaseApp.json:26]: May also be that 6.5 requires a newer ffmpeg.

Anyhow i think "internal ffmpeg is used" is good enough, if -DQT_FEATURE_webengine_system_ffmpeg=OFF does that no opposition from my side.

Comment (Erick555) [reply io.qt.qtwebengine.BaseApp.json:26]: > So if it works in 6.4 then it works in 6.5

The webengine commit which blocks ffmpeg >= 5 (the problem is runtime ffmpeg is too new not too old) was introduced in [6.5.x](https://invent.kde.org/qt/qt/qtwebengine/-/commit/a09ebdd1a722643384b40bdc177a60ee74a30390#) branch. I don't know if 6.4 is silently broken or not affected.

Using internal ffmpeg should be just fine.

Comment (JakobDev) [reply io.qt.qtwebengine.BaseApp.json:26]: As Erik555 said, it doesn't build with `-DQT_FEATURE_webengine_system_ffmpeg=ON`, so we need to disable that.

It is also highly possible, that this is also broken in the 6.4 branch. I assume they got a bug report, that it is not working, so they added this check.

**Comment by JakobDev** [comment]
Is there anything still prevent this from being merged?

**Comment by tsdgeos** [comment]
> Is there anything still prevent this from being merged?

Nobody has said yes to my "Anyhow i think "internal ffmpeg is used" is good enough, if -DQT_FEATURE_webengine_system_ffmpeg=OFF does that no opposition from my side." yet.

Or i missed it ^_^

**Comment by Erick555** [comment]
yes, `-DQT_FEATURE_webengine_system_ffmpeg=OFF` enables internal ffmpeg.

**Comment by tsdgeos** [comment]
"I can't even say if this this even builds"

"Is there anything still prevent this from being merged?"


This is giving me a bit of conflicting information. Have you confirmed this at least builds or have you not? If you have not, what's preventing you to do it?

**Comment by JakobDev** [comment]
> This is giving me a bit of conflicting information. Have you confirmed this at least builds or have you not?

It works. The buildbot has made a successfully testbuild. Did you miss the comment?

**Comment by tsdgeos** [comment]
Ok, i was hoping for something more than "it compiles" but i guess we could start with that...

I just realized i can't commit here though :D

**Comment by hfiguiere** [comment]
let me create the branch then.

**Comment by hfiguiere** [comment]
Branch created.

**Pull request closed without merging**